src/components/footer.js

```diff
@@ -52,16 +52,11 @@ function Footer() {
             size="medium"
             image={
               <img
-                className="usa-footer__logo-img"
```

Comment:
How come the class was removed?

Reply:
It made the logo too small - I ended up creating a new className today